x-pack/plugins/index_management/public/application/sections/home/template_list/template_details/tabs/tab_summary.tsx

```diff
@@ -21,84 +25,184 @@ interface Props {
-                return (
-                  <li key={`${indexName}-${i}`}>
-                    <EuiTitle size="xs">
-                      <span>{indexName}</span>
-                    </EuiTitle>
-                  </li>
-                );
-              })}
-            </ul>
-          </EuiText>
-        ) : (
-          indexPatterns.toString()
-        )}
-      </EuiDescriptionListDescription>
+    <EuiFlexGroup data-test-subj="summaryTab">
+      <EuiFlexItem>
+        <EuiDescriptionList textStyle="reverse">
+          {/* Index patterns */}
+          <EuiDescriptionListTitle>
+            <FormattedMessage
+              id="xpack.idxMgmt.templateDetails.summaryTab.indexPatternsDescriptionListTitle"
+              defaultMessage="Index {numIndexPatterns, plural, one {pattern} other {patterns}}"
+              values={{ numIndexPatterns }}
+            />
+          </EuiDescriptionListTitle>
+          <EuiDescriptionListDescription>
+            {numIndexPatterns > 1 ? (
+              <EuiText>
+                <ul>
+                  {indexPatterns.map((indexName: string, i: number) => {
+                    return (
+                      <li key={`${indexName}-${i}`}>
+                        <EuiTitle size="xs">
+                          <span>{indexName}</span>
+                        </EuiTitle>
+                      </li>
+                    );
+                  })}
+                </ul>
+              </EuiText>
+            ) : (
+              indexPatterns.toString()
+            )}
+          </EuiDescriptionListDescription>
+
+          {/* Priority / Order */}
+          {isLegacy !== true ? (
+            <>
+              <EuiDescriptionListTitle>
+                <FormattedMessage
+                  id="xpack.idxMgmt.templateDetails.summaryTab.priorityDescriptionListTitle"
+                  defaultMessage="Priority"
+                />
+              </EuiDescriptionListTitle>
+              <EuiDescriptionListDescription>
+                {priority || priority === 0 ? priority : i18nTexts.none}
+              </EuiDescriptionListDescription>
+            </>
+          ) : (
+            <>
+              <EuiDescriptionListTitle>
+                <FormattedMessage
+                  id="xpack.idxMgmt.templateDetails.summaryTab.orderDescriptionListTitle"
+                  defaultMessage="Order"
+                />
+              </EuiDescriptionListTitle>
+              <EuiDescriptionListDescription>
+                {order || order === 0 ? order : i18nTexts.none}
+              </EuiDescriptionListDescription>
+            </>
+          )}
+
+          {/* Components */}
+          {isLegacy !== true && (
+            <>
+              <EuiDescriptionListTitle data-test-subj="componentsTitle">
+                <FormattedMessage
+                  id="xpack.idxMgmt.templateDetails.summaryTab.componentsDescriptionListTitle"
+                  defaultMessage="Components"
```

Comment:
I'd change this to `Component templates`.

I consider components to be the actual mappings, settings, and aliases that get created on the target index/data stream.

Reply:
Ok. I will then update the creation flow to match this. I thought it was verbose to have "Component template" everywhere, but let's be consistent.

<img width="1188" alt="Screenshot 2020-07-08 at 10 36 37" src="https://user-images.githubusercontent.com/2854616/86900496-b5f8bd80-c10b-11ea-94ee-1bcce3be917d.png">